web/README.md

```diff
@@ -14,33 +26,58 @@ link your build folder (`dist`) or just rely on the `devel-install` task:
     make devel-install
 ```
 
+Then you can visit the D-installer module through the following URL:
+
+http://localhost:9090/cockpit/@localhost/d-installer/index.html.
+
 Bear in mind that if something goes wrong while building the application (e.g., the linter fails),
 the link will not be created.
 
-While working on the code, you might want to run the [webpack-dev-server](https://github.com/webpack/webpack-dev-server)
-to get the [Hot Module Replacement](https://webpack.js.org/concepts/hot-module-replacement/) enabled.
+To automatically rebuild the sources after any change you can run
 
 ```
-    npm run server
+    npm run watch
 ```
 
-Unfortunately, it does not work remotely yet because it needs to connect to
-'localhost'. You might workaround it by just using the browser from the VM
-through SSH (e.g., "ssh -X yourmachine firefox" -or chromium-).
+*But do not forget that you have to reload the code in your browser manually after each change!*
 
-Thus, if live reloading does not work for you, using the Webpack watcher for
-refreshing the build every time a file is changed still being an option through
-the command below. But please, do not forget that you have to reload the code
-in your browser manually.
+### Using a development server
 
+To start the [webpack-dev-server](https://github.com/webpack/webpack-dev-server)
+use this command:
 
 ```
-    npm run watch
+    npm run server -- --open
```

Comment:
The `--open` parameter does not work if the regular case. It tries to open `http://localhost:8080` and it does not work. It should open `https://localhost:9090/cockpit/@localhost/d-installer/index.html` (or, at least, `http://localhost:9090`).